spec/presenters/guide_presenter_spec.rb

```diff
@@ -102,27 +119,31 @@
         [edition.content_owner.content_id]
       )
     end
-  end
-end
 
-RSpec.describe GuidePresenter, "for a Point" do
-  it "includes the service standard as a parent in the links" do
-    edition = create(:edition)
-    point = create(:point, editions: [edition])
+    context 'for a guide community' do
+      let(:guide) { create(:guide_community) }
```

Comment:
Overriding `let`s can be confusing. Starting a new `describe` block is nice way of starting fresh. It's another trick for making the reader jump around less.. If you're still keen to use nesting then it's best to move the initial `let` into it's own context.